### PR TITLE
Linux docs changes

### PR DESCRIPTION
**Pull request by cyrv6737**
Main things included in this PR:

1) Edited the wording of the EA App warning to better reflect the actual functionality. Compatibility is not broken if the user has signed into EA App, at least according to my testing.

2) Move all Linux docs to a new section called `SteamDeck and Linux` so users can more easily find installation documentation.

3) Moved troubleshooting to its own page.

4) Added a troubleshooting section about fixing blank EA App under NorthstarProton

I have absolutely no clue how Gitbook handles sidebar structure so I took my best guess, let me know if anything is bad/needs changes.

### CODE REVIEW

File: docs/steamdeck-and-linux/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# SteamDeck and Linux
+
+Northstar is officially supported on SteamDeck and Linux. Check the subpages for installation, usage, and troubleshooting information.
```
Comment (GeckoEidechse) [inline docs/steamdeck-and-linux/README.md:3]: Probably also want to link to it directly ^^

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]
[SUMMARY.md](https://github.com/R2Northstar/NorthstarWiki/blob/main/docs/SUMMARY.md?plain=1) also needs to be updated accordingly, otherwise the pages won't be visible in sidebar.

File: docs/steamdeck-and-linux/README.md (hunk 1 above)
Comment (cyrv6737) [reply docs/steamdeck-and-linux/README.md:3]: What is it you want there to be a direct link to? Each of the subpages? I followed the format of `installing-northstar/using-northstar/README.md`.

**Comment by cyrv6737** [comment]
@GeckoEidechse I've added Protonup-QT and ProtonPlus as valid methods of installing NorthstarProton since both of these PRs have been merged:

https://github.com/Vysp3r/ProtonPlus/pull/23
https://github.com/DavidoTek/ProtonUp-Qt/pull/144

File: docs/steamdeck-and-linux/README.md (hunk 1 above)
Comment (GeckoEidechse) [reply docs/steamdeck-and-linux/README.md:3]: Yeah, I'd just link all of them tbh.

(Sorry for late response, I thought I responded already, whoops)

**Review by GeckoEidechse — COMMENTED** [review]
You know what, Imma just merge this as is, and let you add the links in separate PR @cyrv6737 :P

**Pull request merged**